pages/[filterName]/insights/index.tsx

```diff
@@ -79,32 +78,24 @@ const InsightsHub: WithPageLayout = () => {
       </Link>
 
       <div className="py-1 md:py-4 flex w-full md:mt-5 justify-between items-center">
-        <div>
-          <div className="">
-            <PaginationResults
-              from={page === 1 ? (insightsMeta.itemCount > 0 ? page : 0) : (page - 1) * insightsMeta.limit + 1}
-              to={page * insightsMeta.limit <= insightsMeta.itemCount ? page * insightsMeta.limit : insightsMeta.itemCount}
-              total={insightsMeta.itemCount}
-              entity={"insights"}
-            />
-          </div>
-        </div>
-        <div>
-          <div className="flex items-center gap-4">
-            <Pagination
-              pages={[]}
-              hasNextPage={insightsMeta.hasNextPage}
-              hasPreviousPage={insightsMeta.hasPreviousPage}
-              totalPage={insightsMeta.pageCount}
-              page={insightsMeta.page}
-              onPageChange={function (page: number): void {
-                setPage(page);
-              }}
-              divisor={true}
-              goToPage
-            />
-          </div>
-        </div>
+        <PaginationResults
+          from={page === 1 ? (insightsMeta.itemCount > 0 ? page : 0) : (page - 1) * insightsMeta.limit + 1}
+          to={page * insightsMeta.limit <= insightsMeta.itemCount ? page * insightsMeta.limit : insightsMeta.itemCount}
+          total={insightsMeta.itemCount}
+          entity={"insights"}
+        />
+        <Pagination
+          pages={[]}
```

Comment:
this doesn't seem to be used

Reply:
Yeah, but we need the `goToPage` component which is inside that `Pagination`